app/grandchallenge/cases/models.py

```diff
@@ -292,6 +309,24 @@ class Image(UUIDModel):
     color_space = models.CharField(
         max_length=5, blank=False, choices=COLOR_SPACES
     )
+    patient_id = models.CharField(max_length=64, null=True, blank=True)
+    patient_name = models.CharField(max_length=324, null=True, blank=True)
+    patient_birth_date = models.DateField(null=True, blank=True)
+    patient_age = models.CharField(max_length=4, null=True, blank=True)
```

Comment:
CharField for age? Wouldn't a small positive integer be more appropiate?

Reply:
No the value in DICOM standard is represented with 3 digits and a single character (either "D", "W", "M", "Y"). So a zeropadded 3 digit number and a character to indicate days/weeks/months/years. We can convert it to a small positive integer but I think this will result in some dataloss/uncertainty because the length of month and year is variable

Reply:
Urgh, that really limits the kind of queries that we can do. But, ok, if we need that then we can add a secondary field.